modules/ROOT/pages/maven-reference.adoc

```diff
@@ -0,0 +1,169 @@
+= Maven Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, maven, version control, dependencies, libraries
+
+This page summarizes reference information that helps you work when using Maven with Mule. Refer to xref:using-maven-with-mule.adoc[Maven Support in Mule] for an introduction and overview.
+
+== Deploying Mule Artifacts to a Central Maven Repository
```

Comment:
Wouldn't be better to have this section at the end of the page? That way this article starts with maven information for beginners and then go to a more specific task.

Reply:
I agree, made the change in 1d33a21. Please confirm if it is OK.